disco_aws_automation/disco_ssm.py

```diff
@@ -0,0 +1,175 @@
+"""
+Manage AWS SSM document creation and execution
+"""
+import os
+import logging
+import time
+import json
+
+import boto3
+
+from botocore.exceptions import ClientError
+from . import read_config
+from .resource_helper import throttled_call, wait_for_state_boto3
+from .exceptions import TimeoutError
+
+logger = logging.getLogger(__name__)
+
+
+SSM_DOCUMENTS_DIR = "ssm/documents"
+SSM_EXT = ".ssm"
+SSM_WAIT_TIMEOUT = 5 * 60
+SSM_WAIT_SLEEP_INTERVAL = 15
+DOCUMENT_PREFIX = "asiaq-"
```

Comment:
Sure, I am thinking we can introduce a `[ssm]` section in `disco_aws.ini` and then we can specify the `document_prefix` setting in there. Though I think it can only has one value at a time (e.g.  either `asiaq` or `amplify`, but not both). Any thoughts?